app/controllers/description_controller_helpers.rb

```diff
@@ -111,7 +111,7 @@ def do_merge_description(src, dest, delete_after)
     src_title = src.format_name
 
     # Doesn't have permission to edit destination.
-    if !in_admin_mode? && !dest.is_writer?(@user)
+    if !in_admin_mode? && !dest.writer?(@user)
```

Comment:
This and similar changes fix [Naming/PredicateName offenses](https://docs.rubocop.org/rubocop/1.5/cops_naming.html#namingmethodname).